kafka-1.0.x/src/main/scala/monix/kafka/KafkaConsumerConfig.scala

```diff
@@ -249,6 +249,7 @@ final case class KafkaConsumerConfig(
   observableCommitType: ObservableCommitType,
   observableCommitOrder: ObservableCommitOrder,
   observableSeekToEndOnStart: Boolean,
+  observableSeekToBeginningOnStart: Boolean,
```

Comment:
Will probably change to ADT, I need to decide whether I want to break it or not

Reply:
It would be nice to have this change before final release.